config/vufind/facets.ini

```diff
@@ -358,3 +361,15 @@ custom_filter_field = "vufind"
 ; combination with checkbox facets if you want to exclude values by default
 ; and provide a checkbox that allows them to be included.
 ;inverted_filters[include_books] = '-format:"Book"'
+
+; Exclude filters can be specified to exclude certain filters from showing in hierarchical
+; facets.
+;[ExcludeFilters]
```

Comment:
Do these settings really apply only to hierarchical facets? If so, would it make sense to name the sections accordingly for greater clarity?

Reply:
Yea, tuned them!